packages/plugin-essentials/sources/commands/explain/peerRequirements.ts

```diff
@@ -55,6 +60,7 @@ export default class ExplainPeerRequirementsCommand extends BaseCommand {
     } else {
       return await explainPeerRequirements(project, {
         stdout: this.context.stdout,
+        onlyWarnings: this.warnings ?? false,
```

Comment:
Please set the default value on the flag itself.

Reply:
Fixed in 1a84d42.